src/assembly/extension.xml

```diff
@@ -0,0 +1,22 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+    <id>openrefine-commons-${openrefine.version}</id>
```

Comment:
I think to make sense for folks (who might not even know about a Commons extension or what `commons` even means, maybe this should say `refine-sample-${openrefine.version}` or just `sample-${openrefine.version}` , so that it aligns with the main pom artifactId ?

Reply:
good point, I changed this into a generic name, it doesn't need to reflect the name of the extension